doc/AzurePipelines.md

```diff
@@ -22,16 +22,22 @@ At high level, the CI/CD pipelines do the following:
 It also runs automated tests during the build steps.
 
 ### Pull request runs
-Due to the length of mobile builds, pipelines are configured to behave a little differently when building in a context of **pull request (PR) build validation**. To reduce the build time, some stages and steps are disabled for PR builds. This requires a specific variable called `IsLightBuild` to be set, hence why it is appearing in the pipeline. 
+
+Due to the length of mobile builds, pipelines are configured to behave a little differently when building in a context of **pull request (PR) build validation**.
+To reduce the build time, some stages and steps are disabled for PR builds.
```

Comment:
Is this true?

Reply:
Yes, iOS.

Reply:
Why do you mean by just "iOS"?

Reply:
The iOS stage

Reply:
and the release stages

Reply:
Well obviously we don't release anything when doing PR validation. But are there other build settings that change?

Reply:
And do you have the `IsLightBuild` variable?

Reply:
Yes

Reply:
> Well obviously we don't release anything when doing PR validation. But are there other build settings that change?

Nah it's light build or not and based on environment if I remember correctly

Reply:
In .NET projects, `IsLightBuild` is used to change the build configuration (such as disabling AOT) to speed it up. If the only thing that is different is whether or not to run the deployment stages, I think the `IsLightBuild` variable doesn't make sense.

Reply:
May be the name doesn't but we can still use it, any CI refactoring should be done in another PR right? I just cleaned up the docs.

Reply:
sure